docs/sources/datasources/elasticsearch.md

```diff
@@ -16,7 +16,7 @@ visualize logs or metrics stored in Elasticsearch. You can also annotate your gr
 1. Open the side menu by clicking the Grafana icon in the top header.
 1. In the side menu under the `Dashboards` link you should find a link named `Data Sources`.
 1. Click the `+ Add data source` button in the top header.
-1. Select *Elasticsearch* from the *Type* dropdown.
+1. Select _Elasticsearch_ from the _Type_ dropdown.
```

Comment:
This should be bold and not italics.

Reply:
This is an auto-formatting issue. * and _ are equivalent in markdown. A single character is italic and double is bold. The current docs site has these as italics so this is functionally correct.
![image](https://user-images.githubusercontent.com/12838032/127212772-3f82f0a9-6203-4068-8701-17e510fb293d.png)

I'll revert the unintended changes to the docs from the auto-formatter.

Reply:
You will find examples of correct usage here: https://grafana.com/docs/grafana/latest/getting-started/getting-started/

As you see, there is no use of italics in procedures.

Reply:
Those were italics before my changes, but since we discovered them here, would you like me to make them bold with this PR?

Reply:
Here is an example:

Click **OK** on the prompt, then change your password.

Also, this maybe of help: https://github.com/grafana/grafana/blob/main/contribute/style-guides/documentation-markdown-guide.md#bold-and-emphasis

Thanks!

Reply:
Thanks for the info! I've made those instances bold now.